assets/js/googlesitekit/components-gm2/Button.stories.js

```diff
@@ -61,6 +61,13 @@ DisabledButton.args = {
 	disabled: true,
 };
 
+export const TertiaryButton = Template.bind( {} );
```

Comment:
It's surprising that we don't have VRT scenarios for button with states. ie. hover, active, focused etc. If possible, could you add them for this tertiary and other buttons? If it's too much trouble, we can file a follow up issue to explore this.

Reply:
That is a very good thought, thank you! However, unfortunately, I have almost reached towards the end of the estimate for this issue, so I have opened #8054 to address this. Please let me know if that sounds good, thanks!

Reply:
Sounds good to me, thank you for filing the issue :tada: